data/avro-schema/src/main/resources/avro/net/corda/data/membership/p2p/UnauthenticatedRegistrationRequestHeader.avsc

```diff
@@ -0,0 +1,23 @@
+{
+  "type": "record",
+  "name": "UnauthenticatedRegistrationRequestHeader",
+  "namespace": "net.corda.data.membership.p2p",
+  "doc": "",
+  "fields": [
+    {
+      "name": "protocolVersion",
+      "doc": "Version of the registration protocol.",
+      "type": "int"
+    },
+    {
+      "name": "timestamp",
+      "doc": "Milliseconds since epoch.",
+      "type": "long"
+    },
+    {
+      "name": "key",
+      "doc": "Member's ephemeral key encoded in PEM format.",
```

Comment:
```suggestion
      "doc": "Member's public ephemeral key encoded in PEM format.",
```
Nit picky - but this change just makes it clearer. Although I would imagine most people would assume it's the public key anyway